test/foundry/deploy/15_holograph_registry_tests.t.sol

```diff
@@ -81,11 +71,20 @@ contract HolographRegistryTests is Test {
     bytes32[] memory emptyBytes32Array;
     bytes memory initCode = abi.encode(deployer, emptyBytes32Array);
     holographRegistry.init(initCode);
+
+    // configHash
+    DeploymentConfig memory deployConfig = HelperDeploymentConfig.getERC721(
+      Constants.getHolographIdL1(),
+      vm.getCode("SampleERC721.sol:SampleERC721"),
+      bytes32(0x0000000000000000000000000000000000000000000000000000000000000086),
```

Comment:
maybe this could be a constant, I think it is used for many tests.

Reply:
Done